eox_core/social_tpa_backends.py

```diff
@@ -4,6 +4,7 @@
 import logging
 
 from django.conf import settings
+from django.contrib.auth.models import User
```

Comment:
```suggestion
from django.contrib.auth import get_user_model
```
And the corresponding `User = get_user_model()`

Reply:
ufff right. Thanks